back/engines/free/email_campaigns/app/views/email_campaigns/moderator_digest_mailer/campaign_mail.mjml

```diff
@@ -1,6 +1,16 @@
 
 <mj-wrapper background-color="#F6F6F6" full-width="full-width" padding="40px 0 0">
 
+  <mj-section>
+    <mj-column>
+      <mj-text>
+        <h2 style="padding: 20px 0">
+          <%= event.project_name %>
```

Comment:
This is not displaying in preview. I haven't worked out why yet.

Reply:
I might need to rebuild container / reset db to pick up the change. Will try that (but seems unlikely)